packages/react/src/refs.tsx

```diff
@@ -116,8 +114,8 @@ function createRefStateManager(
   function ensureSubscription(key: string) {
     if (!refSubscriptions[key]) {
       const refValueProvider = dataSource.get(key)
-      refSubscriptions[key] = refValueProvider.subscribe(() => {
-        const didUpdate = setRefValue(key, refValueProvider.get())
+      refSubscriptions[key] = refValueProvider.subscribe((value) => {
```

Comment:
btw for subscriptions I followed the API provided by React useSyncExternalStore. where the event does not contain the data and instead it will call `get` again

https://react.dev/reference/react/useSyncExternalStore

I used to have useSyncExternalStore but our stuff is too complicated 😂

Reply:
Haha! Which one you prefer tho? Our "streams" implementation passes "value" to the handler(), so our lambda passed to subscribe() has access to "value". I guess it boils down to preference, and I am fine with doing it either way!